webui/react/src/pages/TrialDetails/TrialInfoBox.test.tsx

```diff
@@ -0,0 +1,231 @@
+import { render, screen } from '@testing-library/react';
+import UIProvider, { DefaultTheme } from 'hew/Theme';
+import { ConfirmationProvider } from 'hew/useConfirm';
+
+import {} from 'stores/cluster';
+
+import { ExperimentBase, TrialDetails } from 'types';
+
+import TrialInfoBox from './TrialInfoBox';
+
+vi.useFakeTimers();
+const setup = (trial: TrialDetails, experiment: ExperimentBase) => {
+  const view = render(
+    <UIProvider theme={DefaultTheme.Light}>
+      <ConfirmationProvider>
+        <TrialInfoBox experiment={experiment} trial={trial} />
+      </ConfirmationProvider>
+    </UIProvider>,
+  );
+  return { view };
```

Comment:
can we remove `return { view };` and `const view`?

Reply:
done